tests/SpecTests/FunctionalTestCase.php

```diff
@@ -229,8 +223,7 @@ protected function dropTestAndOutcomeCollections(array $testCollectionDropOption
     /**
      * Insert data fixtures into the test collection.
      *
-     * @param array       $documents
-     * @param string|null $collectionName
+     * @param array $documents
```

Comment:
Redundant. If you want to search for `@param array` and cross-reference with array type hints in method signatures, that might reveal other things you can remove.

Reply:
Removed this and a number of other superfluous type annotations.